packages/react-query/src/__tests__/useQueries.test.tsx

```diff
@@ -1233,20 +1233,13 @@ describe('useQueries', () => {
       const { isLoading } = useQueries({
         queries: ids.map((id) => ({
           queryKey: [key, id],
-          queryFn: () => {
-            return new Promise<{
-              id: number
-              title: string
-            }>((resolve, reject) => {
-              if (id === 2) {
-                setTimeout(() => {
-                  reject(new Error('FAILURE'))
-                }, 10)
-              }
-              setTimeout(() => {
-                resolve({ id, title: `Post ${id}` })
-              }, 10)
-            })
+          queryFn: async () => {
+            if (id === 2) {
+              await sleep(10)
+              return Promise.reject(new Error('FAILURE'))
+            }
+            await sleep(10)
+            return Promise.resolve({ id, title: `Post ${id}` })
           },
```

Comment:
This is just a small thing I wanna make like this.

```suggestion
          queryFn: () => sleep(10).then(() => {
            if(id === 2) throw new Error('FAILURE')
            return { id, title: `Post ${id}` }
          }),
```

Reply:
@manudeli I agree with you. I gonna make it simplify. 👍